example/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java

```diff
@@ -1,23 +1,16 @@
 package io.flutter.plugins;
 
-import io.flutter.plugin.common.PluginRegistry;
+import androidx.annotation.Keep;
+import androidx.annotation.NonNull;
```

Comment:
Are these used?

Reply:
The `Keep` and `NonNull` annotations were auto-generated -- I was having trouble getting the older example application to work, so re-created it using `flutter create`. These annotations appear to be used on lines `12` and `14`.